File: MonolithicApp/MonolithicApp/Domain/Customer.cs
namespace MonolithicApp.Domain
{
    public class Customer
    {
        public string Id { get; set; }
        public string Name { get; set; }

        public Customer()
        {
        }

        public Customer(string name)
        {
            Name = name;
        }

        public Customer(string id, string name)
        {
            Id = id;
            Name = name;
        }
    }
}

File: Microservices/CustomerService/src/main/java/com/avans/customerservice/domain/Customer.java
package com.avans.customerservice.domain;

import lombok.Getter;
import lombok.Setter;

public class Customer {
    @Getter @Setter private String id;
    @Getter @Setter private String name;

    public Customer() {
    }

    public Customer(String name) {
        this.name = name;
    }

    public Customer(String id, String name) {
        this.id = id;
        this.name = name;
    }
}
